js/app.js

```diff
@@ -1 +1,364 @@
-console.log('Hello frontend');
+let options = ["x", "o"];
+let choice = options[0];
+
+//collecting data on starting choice
+//to see which box is clicked
+if(document.querySelector('input[name="choice"]')) {
+    document.querySelectorAll('input[name="choice"]').forEach((element) => {
+    element.addEventListener("change", function(event) {
+        let firstChoice = event.target.value;
+        if (firstChoice === "0") {
+            choice = options[0];
+            console.log(options[0]);
+            showWinner1.innerText = "turn: x";
+            
+        } 
+        if (firstChoice === "1") {
+            choice = options[1];
+            console.log(options[1]);
+            showWinner1.innerText = "turn: o";
+        }
+    });
```

Comment:
This should all be tabbed once more because it is contained inside of the .forEach()